static_src/constants.js

```diff
@@ -270,6 +270,10 @@ const userActionTypes = keymirror({
   USER_INVITE_ERROR: null,
   // Action to dismiss an user list notification.
   USER_LIST_NOTICE_DISMISSED: null,
+  // Action when all roles for space user are removed.
```

Comment:
oops i think i told you to put that comment on the wrong constant. i meant the one below should be the `when all`...